tutorials/optical_tactile_plugin.md

```diff
@@ -171,6 +182,86 @@ is set to `true` and this value is too high, it may slow down the simulation.
+number of forces and check what the sensor is looking at. However, if we
+set the value to less than 15, lags can be expected when moving around the
+simulation due to the cost of requesting the `/marker` service too many times.
+The simulation itself may will even have problems when loading.
+
+- `<contacts_resolution>`: Like the previous parameter, a value too low can
+cause the simulation to slow down or even not load at all. The following table
+aims to provide and idea of how this parameter parameter affects the simulation:
+    ______________________________________________________________________
+   | <contacts_resolution> | no. of contacts computed  |  Average RTF (%)  |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |            1          |            441            |          95       |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |          0.5          |            1681           |          90       |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |          0.3          |            4489           |          70       |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |          0.1          |            40200          |          20       |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+    
+
+While the previous experiments had in mind a unique plugin in the simulation,
+it's important to know how many of them could be loaded before the RTF becomes
+unacceptable. Of course, this depends on the parameters of each individual plugin,
+so all of them have been set as follows:
+
+```
+  <visualization_resolution>15</visualization_resolution>
+  <contacts_resolution>1</contacts_resolution>
+  <visualize_forces>false</visualize_forces>
+  <visualize_sensor>true</visualize_sensor>
+  <contact_radius>0.001</contact_radius>
+  <force_radius>0.0002</force_radius>
+  <force_length>0.01</force_length>
+  <extended_sensing>0.001</extended_sensing>
+```
+
+We found the following RTFs:
+    _____________________________________________
+   |    no. of sensors     |    Worst RTF (%)   |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |            1          |         80         |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |            3          |         54         |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |            5          |         40         |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |            7          |         35         |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+   |           10          |          5         |
+    ⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻⎻
+
+Clearly, the most computational expensive feature is the visualization of the
+normal forces. If `<visualize_forces>` is turned on, problems are found when
+running the simulation with 3 or more sensors, and even 2 sensors is a demanding task.
```

Comment:
"problems" is vague and makes the reader question, "what kind of problems? Should I care about those problems, or maybe it doesn't matter to me?" Either state explicitly what those problems are, or simply say a lag is observed.

Re the service, same question as above about MarkerArray type of request.

Reply:
7b40828